draft-ietf-httpbis-header-structure.md

```diff
@@ -520,13 +513,14 @@ Given an integer as input_integer:
 
 Given a float as input_float:
 
-0. If input_float is not a IEEE 754 double precision number, fail serialisation.
-1. Let output be an empty string.
-2. If input_float is less than (but not equal to) 0, append "-" to output.
-3. Append input_float's integer component represented in base 10 using only decimal digits to output; if it is zero, append "0".
-4. Append "." to output.
-5. Append input_float's decimal component represented in base 10 using only decimal digits to output; if it is zero, append "0".
-6. Return output.
+0. If input_float's fractional component has more than six digits of precision, fail serialisation.
```

Comment:
Q: here we use fractional component, above we use fractional part. Is that ok?

Reply:
Good catch.